azkaban/remote.py

```diff
@@ -518,6 +518,34 @@ def get_schedule(self, name, flow):
       )
     return res['schedule']
 
+  def sla(self, schedule, delay, rule='SUCCESS', emails=None, kill=True):
+    """Set schedule's SLA.
+
+    :param schedule: Schedule ID.
+    :param delay: SLA delay (format %H:%M, i.e. 00:45)
+    :param job: Job name (or empty to set SLA on flow).
+    :param rule: Status to check when duration ends `'SUCCESS'` or `'FINISHED'`
+    :param emails: Comma separated list of emails used for alerting
```

Comment:
Would it be possible to make this argument consistent with the `emails` argument in `run_workflow`?